python_modules/libraries/dagster-dg/dagster_dg/docs/app/components/PackageTree.tsx

```diff
@@ -1,24 +1,12 @@
-import {Contents} from '@/util/types';
-
 import getContents from '@/util/getContents';
 import {Suspense} from 'react';
 import PackageTreeInner from '@/app/components/PackageTreeInner';
 
-interface Props {
-  contents: Contents | null;
-}
-
-export default function PackageTree({contents}: Props) {
+export default async function PackageTree() {
+  const contents = await getContents();
   return (
-    <Suspense fallback={<div>Loading...</div>}>
+    <Suspense fallback={<div>Loading…</div>}>
```

Comment:
Are we actually suspending somewhere?

Reply:
If that's the case can we have a more targeted Suspense boundary around the specific piece that needs to be client rendered? Also are you use you need the Suspense boundary? I think you might just need `"use client"` directive?

Reply:
I am not sure, so I'll try removing it.